src/stacks/nestedLogGroup.ts

```diff
@@ -66,12 +66,13 @@ export class NestedLogGroupAlarmsStack extends BaseNestedStack {
     localConf: config.ConfigMetricAlarms,
   ): void {
     const isEnabled = config.configIsEnabled(this.defaultType, metricFilterName, localConf);
+    const suffix = config.configAlarmSuffix(this.defaultType, metricFilterName, localConf);
 
     if (!isEnabled) {
       return;
     }
 
-    const name = `${groupName}-${metricFilterName}`;
+    const name = `${groupName}-${suffix || metricFilterName}`;
```

Comment:
should this be handled also like this `suffix ? `${metricFilterName}-${suffix}` : metricFilterName`, if there are more than one metric filter in a log group?

Reply:
Good catch, fixed